types/stylelint/index.d.ts

```diff
@@ -83,6 +83,13 @@ declare module 'stylelint' {
 		warn(message: string, options?: StylelintWarningOptions): void;
 	};
 
+	export type Formatter = (
+		results: Array<StylelintResult>,
+		returnValue?: StylelintStandaloneReturnValue,
```

Comment:
This second value isn't used in any of stylelint's built-in formatters. However, it is mentioned in the docs, and there's a test that checks for it. So I've included it in the type definition here.

But maybe it's an old argument that can be removed?